include/zylProg.h

```diff
@@ -34,46 +40,61 @@ class zylProg{
 
 class zylProgManager{
 private:
-	int						m_Count = 0;
-	zylProg*				m_pHead=NULL;
-	zylProg*				m_pActive=NULL;
+	static int					g_Count;
+	static zylProg*				g_pHead;
+	static zylProg*				g_pActive;
 public:
-	void					add(zylProg* ptr);
-	void					focus(int index);
-	void					renderPrograms();
-	void					composite(CRGB** fb);
-	void					input(uint8_t x, uint8_t y, uint8_t z);
+	static void					add(zylProg *ptr);
+	static void					focus(int index);
+	static void					renderPrograms();
+	static void					composite(CRGB *fb);
+	static void					input(uint8_t x, uint8_t y, uint8_t z);
 };
 
-void zylProgManager::add(zylProg* ptr){
-	if (m_pHead == NULL){ //first program
-		m_pHead = ptr;
-		m_pActive = ptr;
+
+/* move to cpp *****************/
+
+//init "static members" (i.e. global class variables)
+
+int zylProgManager::g_Count=0;
+zylProg *zylProgManager::g_pHead=NULL;
+zylProg *zylProgManager::g_pActive=NULL;
+
+// implement "static methods" (i.e. global functions)
+
+void zylProgManager::add(zylProg* ptr)
+{
+	if (g_pHead == NULL){ //first program
+		g_pHead = ptr;
```

Comment:
P.S. If you need to execute some code to init zylProgManager (like in a real constructor) you could add it here